frontend/benefit/applicant/src/components/applications/forms/application/step1/companyInfo/CompanyInfo.tsx

```diff
@@ -0,0 +1,179 @@
+import { APPLICATION_FIELDS } from 'benefit/applicant/constants';
+import { TextInput } from 'hds-react';
+import React from 'react';
+import InputMask from 'react-input-mask';
+import Skeleton from 'react-loading-skeleton';
+import { StyledCheckbox } from 'shared/components/forms/fields/styled';
+import FormSection from 'shared/components/forms/section/FormSection';
+
+import { useComponent } from '../extended';
+import SC from './CompanyInfo.sc';
+import useCompanyInfo from './useCompanyInfo';
+
+const CompanyInfo: React.FC = () => {
+  const { getErrorMessage, fields, translationsBase, formik } = useComponent();
```

Comment:
if we name now component hooks with component name, then all "useComponent" should be refactored but can be done later. What might be not a good idea to use one component hook inside other component, otherwise it becomes reusable hook. In that case its not true, those 4 props can be passed as component props from parent, but i am not sure. Have to think about architecture still.

Reply:
and probably it will be new instance of formik and fields, so when trying to save the form we will get the error

Reply:
Yeah, not sure how `formik` will behave 🤔